docs/docs/02-apps/01-transfer/10-ICS20-v1/05-events.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Events
+sidebar_label: Events
+sidebar_position: 5
+slug: /apps/transfer/ics20-v1/events
+---
+
+:::warning
+This document is relevant only for fungible token transfers over channels on v1 of the ICS-20 protocol.
+:::
+
+# Events
+
+## `MsgTransfer`
+
+| Type         | Attribute Key   | Attribute Value |
+|--------------|-----------------|-----------------|
+| ibc_transfer | sender          | \{sender\}      |
+| ibc_transfer | receiver        | \{receiver\}    |
+| ibc_transfer | tokens          | \{jsonTokens\}  |
+| ibc_transfer | memo            | \{memo\}        |
+| ibc_transfer | forwarding_hops | `nil`           |
```

Comment:
Set value to `nil` since forwarding is not supported in v1, but the attribute will still be emitted.